### PR TITLE
Limit of editing of anonymous user in UI.

### LINKED ISSUE #13412: It should not be possible to edit anonymous user or set certain permissions
It should not be possible to set admin, write, or super user access to the anonymous user. Also it should not be possible to edit this user.

When setting "view" access, a big warning should appear that data will be readable publicly.

### PR DESCRIPTION
**Pull request by diosmosis**
Fixes #13412 

image

### CODE REVIEW

**Comment by diosmosis** [comment]
Updated.

**Comment by diosmosis** [comment]
image

**Comment by tsteur** [comment]
Lets maybe make the small note big as well if possible? As it is quite important that they read it.

**Comment by diosmosis** [comment]
image

File: core/Mail/EmailStyles.php [hunk 1]
```diff
@@ -89,6 +89,21 @@ class EmailStyles
      */
     public $brandNameLong;
 
+    /**
```
Comment (tsteur) [inline core/Mail/EmailStyles.php]: are those changes supposed to be in here @diosmosis ?

Comment (diosmosis) [reply core/Mail/EmailStyles.php]: no, not sure how they got in, will remove

**Comment by tsteur** [comment]
Changes work and good to merge... just need to figure out the email changes 👍

**Pull request merged**